### PR TITLE
fix(datasource/docker): Allow ecr and ecr-fips docker registry for AWS.

### PR DESCRIPTION
**Pull request by nan-coupa**
## Changes

AWS ECR should support the following URLs:
* `*.dkr.ecr.us-east-1.amazonaws.com`
* `*.dkr.ecr-fips.us-east-1.amazonaws.com`

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [X] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [X] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by rarkins — DISMISSED** [review]

File: lib/modules/datasource/docker/ecr.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import type { HttpResponse } from '../../../util/http/types';
 import { regEx } from '../../../util/regex';
 import { addSecretForSanitizing } from '../../../util/sanitize';
 
-export const ecrRegex = regEx(/\d+\.dkr\.ecr\.([-a-z0-9]+)\.amazonaws\.com/);
+export const ecrRegex = regEx(/\d+\.dkr\.ecr(-fips)*\.([-a-z0-9]+)\.amazonaws\.com/);
```
Comment (viceice) [inline lib/modules/datasource/docker/ecr.ts:9]:
```suggestion
export const ecrRegex = regEx(/\d+\.dkr\.ecr(-fips)?\.([-a-z0-9]+)\.amazonaws\.com/);
```

**Review by viceice — COMMENTED** [review]
needs lint fix

**Review by rarkins — DISMISSED** [review]

**Review by viceice — COMMENTED** [review]
still needs a prettier fix

**Comment by nan-coupa** [comment]
@viceice I ran prettier to fix the lint issue.

**Review by viceice — APPROVED** [review]

**Pull request merged**